src/net/link_conditioner.rs

```diff
@@ -0,0 +1,38 @@
+//! This module provides means to simulate various network conditions for development. The primary focus is 
+//! for testing applications under adverse conditions such as high packet loss networks, or high latency
+//! networks.
+
+use rand::prelude::*;
```

Comment:
Disclamer: this is my personal opinion on this

but I would rather not use `*`, just explicitly import the function if you can.

Reply:
omg working on it. Like, I just wrote this 10 minutes ago.

Reply:
There is a reason they call me the fastest review in the west 😉

Reply:
YOU LIVE ON THE EAST COAST